A170_E01_Tomlinson_Eva/src/CumulativeSumLoop.java

```diff
@@ -0,0 +1,17 @@
+import java.util.Scanner;
+
+class CumulativeSumLoop {
+  public static void main(String[] args){
+    cumlSum();
```

Comment:
Please give this a more descriptive name.

```suggestion
    sumUserInputs();
```

Reply:
Good idea! I changed the name!